src/npc.cpp

```diff
@@ -2293,5 +2281,17 @@ void npc::add_msg_player_or_npc(game_message_type type, const char *, const char
     va_end(ap);
 }
 
+void npc::add_new_mission( class mission *miss )
```

Comment:
> Extraneous "class" here, no?

It's required at least by my compiler (gcc 4.9.1), probably because the `npc` class also has a member named `mission` and the compiler seems to confuse the member with the class name. Stupid compiler.

> 'return with a token of your kill',

Good idea, I'll see about that.